frida-gum/src/module_map.rs

```diff
@@ -160,6 +160,17 @@ impl ModuleMap {
             Some(ModuleDetails::from_raw(res))
         }
     }
+
+    /// Get an array of the [`ModuleDetails`] which make up this [`ModuleMap`]
+    pub fn modules(&self) -> Vec<ModuleDetails> {
```

Comment:
Might be a good idea to stick to the name used at the C API level so that language-bindings may become auto-generated in the future without needing special mappings. I.e. `values` here instead of `modules`.

Reply:
ok. Will Do...